settings-system.lp

```diff
@@ -72,67 +72,67 @@ mg.include('scripts/pi-hole/lua/settings_header.lp','r')
                             <tbody>
                                 <tr>
                                     <th scope="row">
-                                        <span>DHCPDISCOVER:</span>
+                                        <span title="Client broadcast to locate available servers">DHCPDISCOVER:</span>
                                     </th>
                                     <td id="sysinfo-dhcp-discover">&nbsp;</td>
                                 </tr>
```

Comment:
This is just a suggestion:

Move each `title` attribute to the table cell (`th`) or to the table row (`tr`).

This way, the user doesn't need to position the pointer exactly on top of the text. Hovering the cell (or row) will show the title.

Reply:
I did not do this intentionally at first because I thought adding it as title of the numbers might be confusing. No I don't think so any longer and moved the description to the `tr` level.